src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/ToolStripKeyboardHandlingService.cs

```diff
@@ -960,18 +953,11 @@ private void OnKeyMove(object sender, EventArgs e)
         {
             // Arrow keys.  Begin a drag if the selection isn't locked.
             ISelectionService selSvc = SelectionService;
-            if (selSvc != null)
+            if (selSvc is not null)
             {
                 MenuCommand cmd = (MenuCommand)sender;
-                if (cmd.CommandID.Equals(MenuCommands.KeySizeWidthIncrease) || cmd.CommandID.Equals(MenuCommands.KeySizeWidthDecrease) ||
-                   cmd.CommandID.Equals(MenuCommands.KeySizeHeightDecrease) || cmd.CommandID.Equals(MenuCommands.KeySizeHeightIncrease))
-                {
-                    _shiftPressed = true;
-                }
-                else
-                {
-                    _shiftPressed = false;
-                }
+                _shiftPressed = cmd.CommandID.Equals(MenuCommands.KeySizeWidthIncrease) || cmd.CommandID.Equals(MenuCommands.KeySizeWidthDecrease) ||
+                   cmd.CommandID.Equals(MenuCommands.KeySizeHeightDecrease) || cmd.CommandID.Equals(MenuCommands.KeySizeHeightIncrease);
```

Comment:
```suggestion
                _shiftPressed = cmd.CommandID.Equals(MenuCommands.KeySizeWidthIncrease)
                    || cmd.CommandID.Equals(MenuCommands.KeySizeWidthDecrease)
                    || cmd.CommandID.Equals(MenuCommands.KeySizeHeightDecrease)
                    || cmd.CommandID.Equals(MenuCommands.KeySizeHeightIncrease);
```

A possible alternative (I _think_ it should work, but it'd require testing):
```suggestion
                _shiftPressed = cmd.CommandID is (MenuCommands.KeySizeWidthIncrease or MenuCommands.KeySizeWidthDecrease or MenuCommands.KeySizeHeightDecrease or MenuCommands.KeySizeHeightIncrease);
```